utils/ErrorOutput.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+)
+
+type ValidationError struct {
+	Field string
+	Error string
+}
+
+func ClientError(status int, body string) (events.APIGatewayProxyResponse, error) {
+
+	resp := events.APIGatewayProxyResponse{
+		Body:       body,
+		StatusCode: status,
+		Headers: map[string]string{
+			"Content-Type": "application/json",
+		},
+	}
+
+	return resp, nil
+}
+func ServerError(err error) (events.APIGatewayProxyResponse, error) {
+	errMsg := ""
+	if err != nil {
+		errMsg = err.Error()
+	}
+	return events.APIGatewayProxyResponse{
+		Body:       errMsg,
```

Comment:
I don't think we should send the exact error messages to users, it could leak important business info and make our applications prone to attacks.

Reply:
Resolved this in this commit of diff PR https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/commits/639e0ae46f9a2b6411fac49779059d9d7c565aca

Reply:
Please link a ticket here

Reply:
Ticket link : #40

Reply:
Ticket link : #40

Reply:
Resolving this @RitikJaiswal75